pkg/network/node/ovscontroller_test.go

```diff
@@ -1138,7 +1138,7 @@ var expectedFlows = []string{
 	" cookie=0, table=100, priority=200, tcp, tcp_dst=53, nw_dst=172.17.0.4, actions=output:2",
 	" cookie=0, table=100, priority=200, udp, udp_dst=53, nw_dst=172.17.0.4, actions=output:2",
 	" cookie=0, table=100, priority=150, ct_state=+rpl, actions=goto_table:101",
-	" cookie=0, table=100, priority=100, reg0=37, ip, actions=ct(commit),move:NXM_NX_REG0[]->NXM_NX_TUN_ID[0..31],set_field:10.0.12.34->tun_dst,output:1",
+	" cookie=0, table=100, priority=100, ip, reg0=37, actions=group:37",
```

Comment:
:frowning_face: read the comment before this array
(you need to increment `ruleVersion` by 1)

Reply:
done